src/ui/public/paginated_table/paginated_table.html

```diff
@@ -28,7 +28,7 @@
       <tbody kbn-rows="page" kbn-rows-min="paginatedTable.rowsToShow(perPage, page.length)"></tbody>
       <tfoot ng-if="showTotal">
         <tr>
-          <th ng-repeat="col in columns" class="numeric-value">{{col.total | number}}</th>
+          <th ng-repeat="col in columns" class="numeric-value">{{col.total}}</th>
```

Comment:
@trevan: why don't you run field formatter on the count ? (count field should provide number field formatter when you call fieldFormatter('html') on the aggConfig)

@epixa i think we do want to remove the number formatting as field formatters should define the format (that's what we do everywhere else)